pyproject.toml

```diff
@@ -17,7 +17,8 @@ cachelib = "*"
 cachetools = "*"
 celery = {extras = ["sqs"], version = "*"}
 click-datetime = "*"
-ddtrace = "*"
+# 24 DEC 2024: Upgrading to the next minor version causes deployments to fail at the "run-deployment/deploy-api" step.
```

Comment:
Don't we have to make a ticket for any dependencies we pin?

Reply:
#2203